examples/platforms/da15000/README.md

```diff
@@ -4,12 +4,42 @@ This directory contains example platform drivers for the [Dialog Semiconductor D
 
 [da15000]: https://support.dialog-semiconductor.com/connectivity/product/openthread-sandbox
 
-**NOTE:** Each Thread node requires a unique EUI-64.  
+**NOTE:** Each Thread node requires a unique EUI-64.
 Please make sure all Thread nodes in your network have a unique EUI-64 by setting HARDCODED_NODE_ID in radio.c to a unique value.
 
 **NOTE:** Current version works only with DA15000 rev. BA
 
-## Build Examples (How to build and flash):
+## Toolchain
+
+Building the examples for the da15000 requires [GNU AutoConf][gnu-autoconf],
+[GNU AutoMake][gnu-automake], [Python][python], and the
+[ARM gcc toolchain][arm-toolchain].
+
+With the exception of the arm toolchain, most of these tools are installed by
+default on modern Posix systems. Windows does not have these tools installed by
+default, and the bootstrap script requires a Posix or MSYS environment to run.
+It is possible to setup an MSYS environment inside of Windows using tools such
+as [Cygwin][cygwin] or [MinGW][mingw] but it is recommended to setup a Linux VM
+for building on a Windows system. For help setting up VirtualBox with Ubuntu,
+consult this [community help wiki article][ubuntu-wiki-virtualbox].
+
+[gnu-autoconf]: https://www.gnu.org/software/autoconf
+[gnu-automake]: https://www.gnu.org/software/automake
+[python]: https://www.python.org
```

Comment:
The python requirement was specifically for `cc2538-bsl.py` using the CC25xx platforms.